lib/components/Keyboard/KeyboardInput/keyboardAccessoryView.api.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "KeyboardAccessoryView",
+  "category": "keyboard",
```

Comment:
extends: `KeyboardTrackingView`

Reply:
It technically does, but for the user it is not true (all `KeyboardTrackingView` props are passed literally in the `KeyboardAccessoryViewProps` and there's no `extends` \ `others` used).

Reply:
Right... I'm not sure why thought